src/argilla/server/database.py

```diff
@@ -27,7 +27,7 @@
     from sqlalchemy.orm import Session
 
 ALEMBIC_CONFIG_FILE = os.path.normpath(os.path.join(os.path.dirname(argilla.__file__), "alembic.ini"))
-TAGGED_REVISIONS = OrderedDict({"1.7": "1769ee58fbb4", "1.8": "ae5522b4c674"})
+TAGGED_REVISIONS = OrderedDict({"1.7": "1769ee58fbb4", "1.8": "ae5522b4c674", "1.10": "3ff6484f8b37"})
```

Comment:
Probably this will change to 1.11

Reply:
Maybe we can have this track in the CHANGELOG as a change instead of supporting this revision tagging. Let's discuss it.